lib/internal/assert/assertion_error.js

```diff
@@ -313,6 +313,17 @@ function createErrDiff(actual, expected, operator) {
   return `${msg}${skipped ? skippedMsg : ''}\n${res}${other}${end}${indicator}`;
 }
 
+function addEllipsis(string) {
+  const lines = StringPrototypeSplit(string, '\n', 11);
```

Comment:
@aduh95 Thanks for fixing the bug. Why did we add `11` here?

Reply:
We only want to keep the first 10 lines, we can give up splitting the string at the 11th line.